### PR TITLE
LPS-149384, LPS-149369 | Master

### PR DESCRIPTION
**Pull request by Jeremy-Ch**
https://issues.liferay.com/browse/LPS-149384
https://issues.liferay.com/browse/LPS-149369

/cc @[chaowang-123](https://github.com/chaowang-123) @[jaydawu](https://github.com/jaydawu)

### CODE REVIEW

**Comment by Jeremy-Ch** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/documentmanagement/documentsadministration/DMLocalStaging.testcase [hunk 1]
```diff
@@ -49,8 +49,6 @@ definition {
 	@description = "This is a test for LPS-91043. It asserts that user can not publish an old file entry version to live."
 	@priority = "5"
 	test CanNotPublishOldVerion {
-		property portal.acceptance = "true";
```
Comment (austinchiang) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/documentmanagement/documentsadministration/DMLocalStaging.testcase]: @chaowang-123 we shouldn't add new tests to acceptance by default

Comment (chaowang-123) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/documentmanagement/documentsadministration/DMLocalStaging.testcase]: ok, got it, thanks!

**Comment by austinchiang** [comment]
**Reviewing...** :hourglass_flowing_sand: :sunglasses: :headphones:

**Comment by austinchiang** [comment]
See https://github.com/liferay-lima/liferay-portal/pull/2944

**Pull request closed without merging**